tests/unit/sagemaker/jumpstart/constants.py

```diff
@@ -373,6 +373,232 @@
         "training_enable_network_isolation": False,
         "resource_name_base": "dfsdfsds",
     },
+    "model-artifact-variant-model": {
+        "model_id": "pytorch-ic-mobilenet-v2",
+        "url": "https://pytorch.org/hub/pytorch_vision_mobilenet_v2/",
+        "version": "1.0.0",
+        "min_sdk_version": "2.49.0",
+        "training_supported": True,
+        "incremental_training_supported": True,
+        "hosting_ecr_specs": {
+            "framework": "pytorch",
+            "framework_version": "1.5.0",
+            "py_version": "py3",
+        },
+        "hosting_instance_type_variants": {
+            "regional_aliases": {
+                "us-west-2": {
+                    "gpu_image_uri": "763104351884.dkr.ecr.us-west-2.amazonaws.com/"
+                    "huggingface-pytorch-inference:1.13.1-transformers4.26.0-gpu-py39-cu117-ubuntu20.04",
+                    "cpu_image_uri": "867930986793.dkr.us-west-2.amazonaws.com/cpu-blah",
+                }
+            },
+            "variants": {
+                "p2": {
+                    "regional_properties": {"image_uri": "$gpu_image_uri"},
+                    "properties": {"prepacked_artifact_key": "hello-world-1"},
+                },
+                "p3": {"regional_properties": {"image_uri": "$gpu_image_uri"}},
+                "p4": {"regional_properties": {"image_uri": "$gpu_image_uri"}},
+                "g4dn": {"regional_properties": {"image_uri": "$gpu_image_uri"}},
+                "m2": {"regional_properties": {"image_uri": "$cpu_image_uri"}},
+                "c2": {"regional_properties": {"image_uri": "$cpu_image_uri"}},
+                "ml.g5.48xlarge": {
+                    "properties": {"environment_variables": {"TENSOR_PARALLEL_DEGREE": "8"}}
+                },
+                "ml.g5.12xlarge": {
+                    "properties": {"environment_variables": {"TENSOR_PARALLEL_DEGREE": "4"}}
```

Comment:
could you add a test with an instance specific `prepacked_artifact_key` as well please?

As in not just instance family variant please.

Same for training case please